server/api/rounds.py

```diff
@@ -528,22 +564,47 @@ def draw_sample_for_contest(contest: Contest, sample_size: int) -> List[BallotDr
 
         # Create the pool of ballots to sample (aka manifest) by combining the
         # manifests from every jurisdiction in the contest's universe.
-        # Audits must be deterministic and repeatable for the same real world
-        # inputs. So the sampler expects the same input for the same real world
-        # data. Thus, we use the jurisdiction name and batch keys
-        # (deterministic real world ids) instead of the jurisdiction and batch
-        # ids (non-deterministic uuids that we generate for each audit).
-        manifest = {
-            (jurisdiction.name, batch.tabulator, batch.name): list(
-                range(1, batch.num_ballots + 1)
-            )
-            for jurisdiction in contest.jurisdictions
-            for batch in jurisdiction.batches
-        }
+        if election.audit_type == AuditType.BALLOT_POLLING:
```

Comment:
Do we have to worry about batch comparison too?

Reply:
it's handled in a separate `sample_batches` function